schema/frogbot-schema.json

```diff
@@ -97,6 +97,19 @@
         "description": "Set to true to fail the job if security issues were found.",
         "title": "Fail on Security Issues"
       },
+      "minSeverityFilter": {
+        "type": "string",
+        "default": ["Show all severities"],
+        "description": "Set the minimum severity of vulnerabilities to be fixed and commented on in pull requests.",
+        "title": "Minimum vulnerability severity to filter",
+        "examples": ["low, medium, high, critical"]
+      },
+      "fixableOnly": {
+        "type": "boolean",
+        "default": ["false"],
+        "description": "Display vulnerabilities that has a fixing version only.",
```

Comment:
Display vulnerabilities that has a fixing version only.
-->
Handle vulnerabilities with fix versions only.

Reply:
done